src/lib/Components/Bidding/Components/Input.tsx

```diff
@@ -19,13 +22,17 @@ export class Input extends Component<InputProps, InputState> {
     }
   }
 
-  componentWillReceiveProps(nextProps) {
-    this.setState({
-      borderColor: nextProps.error ? "red100" : "black10",
-    })
+  componentDidUpdate(prevProps) {
```

Comment:
`componentWillReceiveProps` has been deprecated (and was firing an exorbitant amount of times). This works well and seems to be the preferred way.